module-4/backend/week_one.md

```diff
@@ -5,18 +5,69 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+learning about constructor functions and how those work
+
 2. What are some tools to help debug JavaScript code?
+
+debugger, console.log(something), broswer tools cmd + alt + i
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+chai, mocha, 
+
 4. What is the syntax for invoking a function?
+
+function ()
+
 5. What is CORS?
+
+cross origin resource sharing. it is a standard for accessing web resources on different domains. 
+
 6. How do we enable CORS?
+
```

Comment:
how do we enable this in a Rails application specifically though? Or in any other backend application?